core/classifiers/TextClassifier/TextClassifier.py

```diff
@@ -35,6 +35,10 @@ class TextClassifier(base.BaseClassifier):
     Support Vector Classifier (SVC) to obtain the best model using the linear
     kernel.
     """
+
+    NAME_IN_JOB_RESULT_PROTO = 'text_classifier'
+    TYPE_IN_JOB_RESULT_PROTO = 'TextClassifier'
```

Comment:
Are either of these constants being used?

Reply:
No, I converted them to `@property` methods. I will remove these.